<div class='content'>
	<h2>二代测序基础</h2>
	<p><a href="http://www.tudou.com/programs/view/NUYVq0eMRas/" class="external text" target="_blank">陈巍学基因</a>  系列视频。</p>
	
		
	
	
	
	<h2>课题组</h2>
	<p>做alternative cleavage and polyadenylation (APA)研究的: <a href="http://exon.umdnj.edu/lab_home/index.html" target="_blank">Bin Tian 课题组</a> | </p>
	<p><a href="http://blog.sina.com.cn/s/articlelist_1728333092_0_1.html" target="_blank">生信博客</a> 现在已经转为 http://qinqianshan.com/</p>
	<p><a href="http://liulab.dfci.harvard.edu/" target="_blank">X. Shirley Liu Lab</a> - 哈佛大学 刘小乐课题组</p>

	<p><a href="http://www.bio-info-trainee.com/" target="_blank">生信菜鸟团博客</a>，<a href="http://www.biotrainee.com/" target="_blank">生信技能树论坛</a>，微信（biotrainee）：生信菜鸟团：生物信息学学习资料分析，常见数据格式及公共数据库资料分享。常见分析软件及流程，基因检测及癌症相关动态。</p>
	<p><a href="http://guangchuangyu.github.io/" target="_blank">余光创</a>，Y叔在我还没进入生物信息学领域的时候就已经名驰海外(让我印象最深的就是他博客里面的世界地图，满满的各种国旗！)，他的<a href="https://github.com/GuangchuangYu" target="_blank">github</a>, 他的<a href="https://guangchuangyu.github.io/cn/" target="_blank">个人博客</a>。</p>
<pre>
12个可视化包： http://guangchuangyu.github.io/
简历： https://guangchuangyu.github.io/resume/
豆瓣读书： http://guangchuangyu.github.io/cn/douban/
关注Y叔微信公众账号biobabble

编程问题集合： http://www.codeabbey.com/index/task_list
生信微信号集合： http://www.360doc.com/content/17/0305/22/19913717_634279918.shtml 
蛋白PPI绘图：http://mp.weixin.qq.com/s?__biz=MzI4NzExMjU0Mw==&mid=2651105971&idx=1&sn=63a4c92b7c65e2e26d6e6982e0309a47&scene=21#wechat_redirect 

</pre>
	
	
	
	
	
	
	<h2>数据源</h2>
	<p>参考基因组 
<a href="http://support.illumina.com/sequencing/sequencing_software/igenome.html" target="_blank">igenome</a>:下载很慢很难<br>
	<ol>
		<li><a href="ftp://igenome:G3nom3s4u@ussd-ftp.illumina.com/Homo_sapiens/UCSC/hg38/Homo_sapiens_UCSC_hg38.tar.gz" target="_blank">Homo_sapiens_UCSC_hg38</a>[16006984068byte=14.907Gb],</li>
		<li><a href="ftp://igenome:G3nom3s4u@ussd-ftp.illumina.com/Mus_musculus/NCBI/GRCm38/Mus_musculus_NCBI_GRCm38.tar.gz" target="_blank">Mus_musculus_NCBI_GRCm38</a>[29516673107byte=27.48Gb],</li>
		<li><a href="ftp://igenome:G3nom3s4u@ussd-ftp.illumina.com/Homo_sapiens/UCSC/hg19/Homo_sapiens_UCSC_hg19.tar.gz" target="_blank">Homo_sapiens_UCSC_hg19</a>[45468620403byte=42.34Gb],</li>
	</ol>
</p>



	<p>TCGA:<a href="http://cancergenome.nih.gov/" target="_blank">TCGA</a>:The Cancer Genome Atlas - Cancer Genome</p>
	
	<p>Gene Expression Omnibus, <a href="https://www.ncbi.nlm.nih.gov/geo/" target="_blank">GEO</a>(<a href="https://www.ncbi.nlm.nih.gov/geo/browse/" target="_blank">GEO最新数据</a>):GEO is a public functional genomics data repository supporting MIAME-compliant data submissions. Array- and sequence-based data are accepted. Tools are provided to help users query and download experiments and curated gene expression profiles.</p>
	
	<p>Genome Browser Gateway:<a href="https://genome.ucsc.edu/cgi-bin/hgGateway?db=hg38" target="_blank">UCSC</a>:The GRCh38 assembly is the first major revision of the human genome released in more than four years. As with the previous GRCh37 assembly, the Genome Reference Consortium (GRC) is now the primary source for human genome assembly data submitted to GenBank. Beginning with this release, the UCSC Genome Browser version numbers for the human assemblies now match those of the GRC to minimize version confusion. Hence, the GRCh38 assembly is referred to as "hg38" in the Genome Browser datasets and documentation. For a glossary of assembly-related terms, see the GRC Assembly Terminology page.</p>
	<p>调控元件数据库:<a href="http://dnase.genome.duke.edu/" target="_blank">Regulatory Elements Database</a>:This database provides a user interface to the results of Sheffield et al. (2013). The data is from hundreds of DNaseI-hypersensitivity and Affymetrix microarray experiments.</p>
	
	<p><a href="http://gdac.broadinstitute.org/" target="_blank">gdac.</a> @ broadinstitute</p>

	
	
	
	
	
	<h2>数据分析工具</h2>
	<p><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" class="external text" target="_blank">bowtie2文档</a> RNA-seq比对工具。</p>
	<p><a href="http://xena.ucsc.edu/getting-started/" class="external text" target="_blank">UCSC Xena</a> Securely analyze and visualize your private functional genomics data set in the context of public and shared genomic/phenotypic data sets.</p>
	
	

<pre>
1.先学习流程

必定会遇到的三个问题
	1)在linux上安装和使用软件(Linux模块)
	2)需要对数据进行筛选(程序模块)
	3)写文章要画图(R语言模块)
-
一些免费课程：http://shouzhiquan.com/index.php/Index/videos

学习生信，常走的弯路：
	1.门类太广无从下手
	2.书本太理论，看完忘光
	3.不会linux命令
	4.不会编程

课程设计：
	1.生信常用linux命令
	2.编程语言 Python3/perl
	3.画图语言R
	4.可视化工具IGV
	4.项目实战

生物信息学小门类：
	转录组
	变异检测
	芯片数据分析
	宏基因组
	比较基因组
	蛋白质结构预测
	分子模拟
	三维基因组
	更多分析：http://shouzhiquan.com/index.php/Index/price

已经入门，想提高，只能不停地实践！
入门的好方法：找一篇高分论文，照着里面的做，原封不动的还原出来，理解参数设置的意义。过程中啥不会补啥。
</pre>


</div>
